website/content/usage/executors/http.md

```diff
@@ -17,6 +17,9 @@ timeout: Request timeout, unit seconds
 expectCode: Expect response code, such as 200,206
 expectBody: Expect response body, support regexp, such as /success/
 debug: Debug option, will log everything when this option is not empty
+tls_noverifypeer: false (default) or true. If true, disables verification of the remote SSL certificate's validity.
```

Comment:
Great PR =) I have only one proposal to make arguments names more canonical (E.g. istio, kiali, openssl etc.):
```
insecure_skip_verify (instead of tls_noverifypeer)
client_cert (instead of tls_certificatefile)
client_key (instead of tls_certificatekeyfile)
```
Do we also need to have ability to configure root CA cert?

Reply:
@yvanoers Ok, agree about names=) I was adding custom RootCA to docker image root CA, to have ability to use self signed certificates, so probably it will be useful.

Reply:
@Demonian Ok, I will see what I can do to add an option for the Root CA.